scripts/dotnet-cli-build/Utils/NuGetRidUtils.cs

```diff
@@ -0,0 +1,37 @@
+using Microsoft.DotNet.Cli.Build.Framework;
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using NuGet.RuntimeModel;
+
+namespace Microsoft.DotNet.Cli.Build
+{
+    public class NuGetRidUtils
```

Comment:
@krwq I was seeing that our dummy package host was getting published in some of the tests and causing failures. 

This utility uses the NuGet Apis to determine what is and isn't a compatible rid. We only generate a stub package if the dummy package rid is not compatible with the host rid. Since we generate the real package for the host rid I think this makes sense.